content/synthetics/browser_check.md

```diff
@@ -0,0 +1,139 @@
+4. Define on which device to run your check. Available devices are: `Laptop Large`, `Tablet`, and `Mobile Small`.
+5. Pick-up locations to run the test from. Available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+6. Choose a Check frequency between "1 run per 5 minute interval" to "1 run per week":
+
+    {{< img src="synthetics/browser_check/check_frequency.png" alt="Check frequency" responsive="true" style="width:80%;">}}
+
+### Notifications
+
+A notification is sent if at least one step of the Browser Check scenario fails. To configure your notifications:
+
+{{< img src="synthetics/browser_check/browser_check_notification.png" alt="Browser check notification" responsive="true" style="width:80%;">}}
+
+1. Select users and/or [services][1] to send the notifications to.
+2. Enter a **message** for the Browser Check. This field allows standard [Markdown formatting][2]. Notification messages include the **message** defined in this section and information about which assertion failed and why.
+3. Click **Save Details and Record Test** to save your Browser Check.
+4. Start to record your test.
+
+## Record test
+
+**Tests can be only recorded from [Google Chrome][3]. To record your test, download the [Datadog Record Test extension for Google Chrome][4].**
+
+{{< img src="synthetics/browser_check/browser_check_record_test.png" alt="Browser check record test" responsive="true" >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+2. Click on **Start recording** to begin recording your Browser Check.
+3. Your actions are recorded and used to create steps within your Browser Check Scenario.
+4. Use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_check/browser_check_assertions.png" alt="Browser Check assertions" responsive="true" style="width:80%;">}}
+
+    **Note**: **Your last browser check step must be an Assertion**, otherwise there is nothing to check.
+5. Once you have finished your Scenario, click on **Save and Launch Test**.
+
+### Actions
+#### Assertion
+
+{{< img src="synthetics/browser_check/assertions_browser_check.png" alt="Assertion browser check" responsive="true" style="width:40%;">}}
+
+Assertions allow you to check if an element, a content, or some text is available in the current page:
+
+| Assertion | Description |
+| ----      | ----        |
+| `Assert that an element is present on the page` | Asserts that an element is present on the page. This checks if a specific `span`, `div`, `h`, `a`, etc. is somewhere on the current page. |
+| `Check an element's content` |  Makes sure that a specific element is located or not on the current page. |
```

Comment:
From the description here, it seems like this isn't different from "Assert that an element is present on the page." Perhaps it should be "Asserts that the content of a specific element is present on the current page."